### ISSUE TITLE
I18n::Tasks::CommandError: [...] undefined method `to_range' for nil:NilClass

### ISSUE THREAD

**Issue by cbeer** [glebm/i18n-tasks#426]
With v1.0.1, I'm seeing this error:

```
 # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/local_ruby_parser.rb:57:in `updated_location'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/local_ruby_parser.rb:45:in `normalize_location'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/local_ruby_parser.rb:44:in `block in normalize_location'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/local_ruby_parser.rb:44:in `map'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/local_ruby_parser.rb:44:in `normalize_location'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/local_ruby_parser.rb:27:in `parse'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/erb_ast_processor.rb:22:in `on_code'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/ast-2.4.2/lib/ast/processor/mixin.rb:259:in `process'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/erb_ast_processor.rb:41:in `block in handler_missing'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/erb_ast_processor.rb:41:in `map'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/erb_ast_processor.rb:41:in `handler_missing'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/ast-2.4.2/lib/ast/processor/mixin.rb:261:in `process'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/erb_ast_processor.rb:41:in `block in handler_missing'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/erb_ast_processor.rb:41:in `map'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/erb_ast_processor.rb:41:in `handler_missing'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/ast-2.4.2/lib/ast/processor/mixin.rb:261:in `process'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/erb_ast_processor.rb:41:in `block in handler_missing'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/erb_ast_processor.rb:41:in `map'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/erb_ast_processor.rb:41:in `handler_missing'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/ast-2.4.2/lib/ast/processor/mixin.rb:261:in `process'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/erb_ast_processor.rb:17:in `process_and_extract_comments'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/erb_ast_scanner.rb:25:in `path_to_ast_and_comments'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/ruby_ast_scanner.rb:35:in `scan_file'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/file_scanner.rb:27:in `block in keys'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/files/file_finder.rb:30:in `map'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/files/file_finder.rb:30:in `traverse_files'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/file_scanner.rb:57:in `traverse_files'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/file_scanner.rb:26:in `keys'
     # /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/bundler/gems/i18n-tasks-dbbcd3abe659/lib/i18n/tasks/scanners/scanner_multiplexer.rb:34:in `block (3 levels) in collect_results'
```

From parsing a file like:

```
<%= render Blacklight::System::ModalComponent.new do |component| %>
  <% component.title { t('blacklight.tools.citation') } %>

  <%= render Blacklight::Document::CitationComponent.with_collection(@documents) if @documents.present? %>
<% end %>
```

**Comment by cbeer** [comment]
Yes, it is.

> /Users/cabeer/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/i18n-tasks-1.0.2/lib/i18n/tasks/scanners/local_ruby_parser.rb:57:in `updated_location': Error scanning app/views/catalog/_citation.html.erb: undefined method `to_range' for nil:NilClass (I18n::Tasks::CommandError)

**Comment by glebm** [comment]
@davidwessman Can you please have a look?

**Comment by glebm** [comment]
Released v1.0.3

**Comment by cbeer** [comment]
Thank you!
